src/proto/ctl/system.proto

```diff
@@ -73,7 +73,7 @@ message SystemStartResp {
 
 // SystemQueryReq supplies system query parameters.
 message SystemQueryReq {
-	int32 rank = 1; // negative indicates NilRank
+	repeated uint32 ranks = 1; // system ranks to query
```

Comment:
Do we want to be careful about changing PB APIs this late in the game? After 1.0 we'll probably expect these to stabilize.

Reply:
yes I want to get this in for 1.0 because it removes the hack